README.md

```diff
@@ -208,6 +208,9 @@ maba_webpack:
                                         # for example, set dev_server_public_path and public_path to overwrite
                                             # http://localhost:8080/compiled/ and /compiled/
                                             # see inside your webpack.config.js for more info
+        # set location of cached manifests. Useful for deploy, when you don't want to include your cache directory
```

Comment:
@mariusbalcytis: They are not the only ones. This is the same I'd like to do. I don't want node garbage + a zillion unnecessary files on my production servers